allure-rest-assured/src/main/java/io/qameta/allure/restassured/AllureLoggerFilter.java

```diff
@@ -0,0 +1,68 @@
+package io.qameta.allure.restassured;
+
+import io.qameta.allure.curl.CurlBuilder;
+import io.qameta.allure.curl.FreemarkerUtils;
+import io.restassured.filter.Filter;
+import io.restassured.filter.FilterContext;
+import io.restassured.internal.NameAndValue;
+import io.restassured.response.Response;
+import io.restassured.specification.FilterableRequestSpecification;
+import io.restassured.specification.FilterableResponseSpecification;
+import ru.yandex.qatools.allure.Allure;
+import ru.yandex.qatools.allure.events.MakeAttachmentEvent;
+import ru.yandex.qatools.allure.events.StepFinishedEvent;
+import ru.yandex.qatools.allure.events.StepStartedEvent;
+
+import java.util.HashMap;
+import java.util.Map;
+
+public class AllureLoggerFilter implements Filter {
+
+    public static AllureLoggerFilter log() {
+        return new AllureLoggerFilter();
+    }
+
+    @Override
+    public Response filter(FilterableRequestSpecification requestSpec, FilterableResponseSpecification responseSpec,
+                           FilterContext filterContext) {
+
+        AllureData allureData = new AllureData();
+        Response response = filterContext.next(requestSpec, responseSpec);
+
+        String uri = requestSpec.getURI();
+        String method = requestSpec.getMethod();
+        String requestBody = requestSpec.getBody();
+
+        allureData.setRequestUrl(uri);
+        allureData.setRequestMethod(method);
+        allureData.setRequestBody(requestBody);
```

Comment:
According to screens body is not prettified
this code should solve it:
`allureData.setRequestBody(new Prettifier().getPrettifiedBodyIfPossible(requestBody))`
Same for response applicable